cli/src/commands/mod.rs

```diff
@@ -355,7 +355,7 @@ struct StatusArgs {}
 #[derive(clap::Args, Clone, Debug)]
 struct LogArgs {
     /// Which revisions to show. Defaults to the `revsets.log` setting, or
-    /// `@ | ancestors((remote_branches() | tags()).., 2)` if it is not set.
+    /// `(trunk()..@):: | (trunk()..@)-` if it is not set.
```

Comment:
I prefer to see all unmerged branches (i.e. `ancestors(trunk().., 2)`) by default

Reply:
For either option, I'd change it a bit to add a few more revisions:

`ancestors(trunk().., 2) | trunk() | @` in Yuya's case.

`ancestors(trunk()..@, 2)| trunk()` in your case.

### Less actionable thoughts

I have the same preference as Yuya for my personal config, though I'm not sure this is everybody's preference. I think the revset you suggested is also useful. I'm trying out  a `stack` alias for `trunk()..@` and a `logstack` alias for the revset I proposed above. 

I also put some thoughts about how a default revset could work in a world with immutable heads in https://github.com/martinvonz/jj/pull/2247#issuecomment-1732214557.